x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-ebs-overview.json

```diff
@@ -0,0 +1,906 @@
+{
+  "objects": [
+    {
+      "attributes": {
+        "description": "Overview of AWS EBS Metrics",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "optionsJSON": {
+          "hidePanelTitles": false,
+          "useMargins": true
+        },
+        "panelsJSON": [
+          {
+            "embeddableConfig": {},
+            "gridData": {
+              "h": 10,
+              "i": "1",
+              "w": 24,
+              "x": 24,
+              "y": 10
+            },
+            "panelIndex": "1",
+            "panelRefName": "panel_0",
+            "title": "Volume Write Ops",
+            "version": "7.3.0"
```

Comment:
Tests are failing because of these 7.3.0 versions, you will need to update with master so tests pass.

Reply:
Hmmm seems like travis ci is passing but jenkins is not. But I can't really tell the jenkins ci failed because of the version problem. @jsoriano Could you point out the error for me please 😬

Reply:
Oh, you are right, I saw test_dashboards failing and I assumed it was because of that. But I cannot find where I saw it, sorry.

Reply:
That's alright 😬 I'm glad 7.3.0 works 👍 Thanks!!